model/registry/azure.yaml

```diff
@@ -0,0 +1,115 @@
+groups:
+  - id: registry.azure.sdk
+    prefix: az
+    type: attribute_group
+    brief: >
+      This document defines generic attributes for Azure SDK.
+    attributes:
+      - id: service_request_id
+        type: string
+        stability: experimental
+        brief: "The unique reference string that identifies a request"
+        examples:
+          - "00000000-0000-0000-0000-000000000000"
+  - id: az.log
+    stability: experimental
+    type: event
```

Comment:
we should not define events in the registry (it's only for attributes for the time being), please move `az.log` definition to another file in `/model/logs`